eng/Versions.props

```diff
@@ -11,7 +11,7 @@
     <PatchVersion>0</PatchVersion>
     <VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>2</PreReleaseVersionIteration>
+    <PreReleaseVersionIteration>3</PreReleaseVersionIteration>
```

Comment:
Why is this getting mvoed to 3 now (vs. when we branch?)

Reply:
I just combined the changes together, we wont be building or shipping anything from this in preview 2.

Reply:
Is it being retired going forward? Or will there be things for preview 3?

Reply:
its being retired going forward. we dont want to ship anything in preview2 as well from extensions repo.